src/Extension/CoreExtension.php

```diff
@@ -296,7 +296,7 @@ public function getOperators(): array
     {
         return [
             [
-                'not' => ['precedence' => 50, 'class' => NotUnary::class],
+                'not' => ['precedence' => 50, 'future_precedence' => 70, 'class' => NotUnary::class, 'package' => 'twig/twig', 'version' => '3.15'],
```

Comment:
should we have `'future_precedence' => new PrecedenceChangeInfo(70, 'twig/twig', '3.15')` instead, for consistency with the new way to provide deprecations for callables ? The current naming does not make it clear that `version` applies to the precedence deprecation.

Reply:
Good idea